dependencies/deps.go

```diff
@@ -90,6 +84,88 @@ func Minimal(x interface{}) (resolved []ast.Ref, err error) {
 	}), nil
 }
 
+// Base returns the list of base data documents that the given AST element depends on.
```

Comment:
I'd include comment to indicate that the returned refs are always constant (i.e., no vars/nested refs/comprehensions, etc.) E.g., in the future, we may want finer-grained results that include vars.